nba-dashboard/requirements.txt

```
@@ -0,0 +1,3 @@
+pandas
+nba_api
+plotly
```

Should shinywidgets be on this list?

```suggestion
plotly
shinywidgets
```

I'm guessing you're thinking of this from the shinylive perspective where shinywidgets is already available. But I was trying to run this locally and it would have been helpful for `pip install -r requirements.txt` to install all of the requirements. In that regard, adding `shiny` might be helpful too.